project/src/components/room-comment-form/room-comment-form.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+function RoomCommentForm() {
+  const [data, setData] = useState({
+    user: '',
+    rating: 0,
+    review: '',
+  });
+  const {review} = data;
+
+  const handleFieldChange = (evt) => {
+    const {name, value} = evt.target;
+    setData((prev) => ({...prev, [name]: value}))
```

Comment:
```suggestion
  const handleFieldChange = (evt) => {
    const {name, value} = evt.target;
    setData({... data, [name]: value})
```
prevState обычно нужен когда хочешь завязаться на предыдущее состояние 🙂